tests/components/radial-tests.js

```diff
@@ -46,7 +47,7 @@ test('RadialChart: Basic rendering', t => {
 test('RadialChart: Showcase Example - Simple Radial Chart Example', t => {
   const $ = mount(<SimpleRadialChart />);
   t.equal($.find('.rv-radial-chart__series--pie__slice').length, 5, 'should find the same number of slices as data entries');
-  t.equal($.find('.rv-radial-chart__series--pie__slice-overlay').length, 5, 'should find the right number of overlay slices');
```

Comment:
i'd be careful about modifying the overlays, that's a kinda core part of the way the radial chart does mouse event handling